jerry-debugger/jerry-client.py

```diff
@@ -136,16 +146,14 @@ def main():
                            JERRY_DEBUGGER_SOURCE_FILE_NAME,
                            JERRY_DEBUGGER_FUNCTION_NAME,
                            JERRY_DEBUGGER_PARSE_FUNCTION,
-                           JERRY_DEBUGGER_BYTE_CODE_CPTR]:
+                           JERRY_DEBUGGER_BYTE_CODE_CPTR,
+                           JERRY_DEBUGGER_FREE_BYTE_CODE_CPTR]:
```

Comment:
Byte code free should not be handled by the parser, it should be an independent event.